commons-statistics-descriptive/pom.xml

```diff
@@ -56,6 +56,12 @@
             <scope>test</scope>
         </dependency>
 
+        <dependency>
+            <groupId>org.apache.commons</groupId>
+            <artifactId>commons-numbers-core</artifactId>
+            <version>1.2-SNAPSHOT</version>
```

Comment:
This module alone should not use a different version from the entire project. If you have different modules using different versions then a downstream project which transitively includes multiple modules from here may not correctly resolve to the one that this module requires. The change should be made to the property in the parent pom `<statistics.commons.numbers.version>`.